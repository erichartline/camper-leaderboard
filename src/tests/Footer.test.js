import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../components/Footer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
});

test('should render Header correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
