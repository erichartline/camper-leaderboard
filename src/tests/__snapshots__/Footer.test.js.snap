// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="footer"
>
  <p>
    Written and coded by 
    <a
      href="http://www.erichartline.net"
      rel="noopener noreferrer"
      target="_blank"
    >
      Eric Hartline
    </a>
     for 
    <a
      href="http://www.freecodecamp.org/"
      rel="noopener noreferrer"
      target="_blank"
    >
      freeCodeCamp
    </a>
     
    <i
      aria-hidden="true"
      className="fa fa-free-code-camp"
    >
      .
    </i>
  </p>
</div>
`;
